services/management/ia/templates_service.go

```diff
@@ -370,26 +371,19 @@ func (s *TemplatesService) loadTemplatesFromDB() ([]templateInfo, error) {
 func (s *TemplatesService) downloadTemplates(ctx context.Context) ([]alert.Template, error) {
 	s.l.Infof("Downloading templates from %s ...", s.host)
 
-	settings, err := models.GetSettings(s.db)
+	ssoDetails, err := models.GetPerconaSSODetails(ctx, s.db.Querier)
 	if err != nil {
 		return nil, err
 	}
 
-	cc, err := saasdial.Dial(ctx, settings.SaaS.SessionID, s.host)
+	bodyBytes, err := saasdial.Dial(ctx, s.host, ssoDetails.AccessToken.AccessToken)
```

Comment:
Is this going to get us the templates? or do we need to append some endpoint to `s.host`?

Reply:
Because for example on this line https://github.com/percona/pmm-managed/pull/866/files#diff-a0afcd6e7a3e5aaf7ff647e3b665ae863d07c64e7c077b084f880809a4a176e1R1061 we append endpoint.

Reply:
@palash25 PTAL if telemetry and templates does have endpoints, or it match host. Thank you.

Reply:
yes, all of them have different endpoints, so you need to to mention them explicitly otherwise it won't work

you can find them here
https://github.com/percona-platform/platform/blob/main/api/check/retrieval/retrieval_api.proto#L22
https://github.com/percona-platform/platform/blob/main/api/telemetry/reporter/reporter_api.proto#L16

Reply:
Endpoint for telemetry and templates added.